File: src/main/java/com/ambition/business/user/domain/SysGroup.java
package com.ambition.business.user.domain;

import com.baomidou.mybatisplus.annotation.IdType;
import java.util.Date;
import com.baomidou.mybatisplus.annotation.TableId;
import java.io.Serializable;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

/**
 * <p>
 * 后台分组表
 * </p>
 *
 * @author wuys
 * @since 2019-12-11
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
public class SysGroup implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    /**
     * 组名称
     */
    private String groupName;

    /**
     * 描述
     */
    private String description;

    /**
     * 是否有效
     */
    private Integer isValid;

    /**
     * 创建时间
     */
    private Date createTime;

    /**
     * 创建者id
     */
    private Long createUserId;

    /**
     * 更新时间
     */
    private Date updateTime;

    /**
     * 更新者id
     */
    private Long updateUserId;

    /**
     * 更新者用户名
     */
    private String updateUserName;

    /**
     * 创建者用户名
     */
    private String createUserName;


}

File: src/main/java/com/ambition/business/user/domain/SysGroup.kt
package com.ambition.business.user.domain

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import java.util.*

/**
 * <pre>
 *
 *
 * @title: SysGroup
 * @description:
 * @author: wuys
 * @datetime: 2019-12-18 16:49
 * </pre>
 */
data class SysGroup(
        @TableId(value = "id", type = IdType.AUTO)
         var id: Long? = null,

        /**
         * 组名称
         */
         var groupName: String? = null,

        /**
         * 描述
         */
         var description: String? = null,

        /**
         * 是否有效
         */
         var isValid: Int? = null,

        /**
         * 创建时间
         */
         var createTime: Date? = null,

        /**
         * 创建者id
         */
         var createUserId: Long? = null,

        /**
         * 更新时间
         */
         var updateTime: Date? = null,

        /**
         * 更新者id
         */
         var updateUserId: Long? = null,

        /**
         * 更新者用户名
         */
         var updateUserName: String? = null,

        /**
         * 创建者用户名
         */
         var createUserName: String? = null)